qutebrowser/config/configdata.yml

```diff
@@ -1621,9 +1621,23 @@ hints.chars:
     completions:
       - ['asdfghjkl', "Home row"]
       - ['aoeuidnths', "Home row (Dvorak)"]
+      - ['hjkl', "Directional keys (Vim)"]
       - ['abcdefghijklmnopqrstuvwxyz', "All letters"]
   desc: Characters used for hint strings.
 
+hints.labels:
+  default:
+  type:
+    name: UniqueCharString
+    none_ok: true
+    completions:
+      - ['←↓↑→', "Arrows for directional keys (Vim)"]
```

Comment:
You weren't unclear. Maybe I was too quick to mark the conversation as resolved, as I'm not really sure what is the best practise working with these reviews. 

One thing why I'm a bit hesitant to bring the changes back is that currently the labels `←↓↑→↖↗↙↘` and `←↓↑→` are effectively the same thing when using chars `hjkl` or `hjklyubn`. Maybe the only completion for labels should be `←↓↑→↖↗↙↘` and chars could include both `hjkl` or `hjklyubn`? This would have the added benefit of communicating to the user that chars and labels can be of different lengths.

Reply:
I restored the previous extended completions as using one `hints.labels` completion for two `hints.chars` would have been a little confusing